lib/errors.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const errorTitle = (code) => `${code}_ERROR`;
+const errorMsg = (code) => `There is a ${code}_ERROR`;
+
+const getContent = (err, debug, translate) => {
+
+  const content = {};
```

Comment:
Const isn't to do with immutability, it is to do with assignment - if you assign a value to a const you cannot reassign the const to a different value. Immutability is another thing entirely and not applicable in this example. We could use immutable.js if we want to make items immutable but that would be another discussion